### PR TITLE
Fix doc typos

### PR DESCRIPTION
**Pull request by muzimuzhi**
I'm less confident about the change done in second commit.

### CODE REVIEW

**Comment by T-F-S** [comment]
Thank you once again. Currently, I have no time, but I will check it later.

**Comment by T-F-S** [comment]
I merged the commits to download and compare them with my source. I agree to correct the missing typos from the first commit, but not to the changes of the second commit.

You may be right that your proposed changes from the second commit result in better English, but, personally, I understand my version better. So, these changes will be reverted back by my next commit. Nevertheless, thank you for your effort to improve the documentation!

**Comment by muzimuzhi** [comment]
Thanks and totally understandable.

File: doc/latex/csvsimple/csvsimple-l3.tex [hunk 1]
```diff
@@ -723,7 +723,7 @@ \section{Macros for the Processing of CSV Files}\label{sec:makros}%
   Typesets the current data line number. This is the
   current number of accepted data lines without the header line.
   Despite of the name, there is no associated \LaTeX\ counter |csvrow|,
-  but \refCom{thecsvrow} is an accessor the \LaTeX3 integer
+  but \refCom{thecsvrow} being an accessor of the \LaTeX3 integer
   \refCom{g_csvsim_row_int}.
```
Comment (muzimuzhi) [inline doc/latex/csvsimple/csvsimple-l3.tex:727]: For the three similar occurrences:
- the first two seem to lack an "of": is an accessor the LaTeX3 integer -> is an accessor **of** the LaTeX3 integer
- the third one was changed in e32ef0539140d6cfa79caf633fe50f9db1d9842d, hence different from the other two. Maybe you want to adapt the first two similarly.
    ```diff
    @@ -754,7 +755,7 @@ \section{Macros for the Processing of CSV Files}\label{sec:makros}%
       current number of all data lines including the header line and all
       lines filtered out.
       Despite of the name, there is no associated \LaTeX\ counter |csvinputline|,
    -  but \refCom{thecsvinputline} is an accessor the \LaTeX3 integer
    +  but \refCom{thecsvinputline} accesses the \LaTeX3 integer
       \refCom{g_csvsim_inputline_int}.
     \begin{dispExample}
     \csvreader[
    ```

Comment (T-F-S) [reply doc/latex/csvsimple/csvsimple-l3.tex:727]: Yes, you are right. I will adapt the first two accordingly.

Comment (muzimuzhi) [reply doc/latex/csvsimple/csvsimple-l3.tex:727]: Just for the record, the first two were adapted in [v2.5.0](https://github.com/T-F-S/csvsimple/releases/tag/v2.5.0) (3cb41f7927ea815d0341226ac9d43bbecaaaf9f4). Thanks.

**Pull request merged**